lib/compiler.js

```diff
@@ -323,8 +319,15 @@ function resolveSourceDir(rootDir, sourceDir) {
       [0];
     if (srcDir)
       return srcDir;
+  } else {
+    try {
+      var resolved = path.resolve(relativePath);
```

Comment:
Please give `resolved` a better, more descriptive name. E.g. `absolutePath` or `fullPath`.

Add a comment explaining why we need to call `require.resolve`.